src/components/input/input.scss

```diff
@@ -91,6 +91,12 @@ md-input-container {
     bottom: 100%;
     @include rtl(left, 0, auto);
     @include rtl(right, auto, 0);
+
+    &.md-required:after {
+      content: ' *';
```

Comment:
Ah, just looked a bit more; it's actually an issue with the `scale(0.75)` transform.

Looks like we can fix it with `transform-origin: right;`.